### PR TITLE
CVSB-6219

### PR DESCRIPTION
**Pull request by vladddragomir**

### CODE REVIEW

File: src/app/app.enums.ts [hunk 1]
```diff
@@ -300,7 +300,6 @@ export enum PAGE_NAMES {
   TEST_TYPES_LIST_PAGE = 'TestTypesListPage',
   REASONS_SELECTION_PAGE = 'ReasonsSelectionPage',
   TEST_CANCEL_PAGE = 'TestCancelPage',
-  TEST_CREATE_PAGE = 'TestCreatePage',
```
Comment (agoncearenco) [inline src/app/app.enums.ts]: Is this change intended?

Comment (vladddragomir) [reply src/app/app.enums.ts]: no, it was duplicated so i removed it

Comment (vladddragomir) [reply src/app/app.enums.ts]: i mean yes, it was intended
